### ISSUE TITLE
Choco release soon?

### ISSUE THREAD

**Issue by Spunkie** [Cossey/ScreenPixelRuler2#1]
Title says it all really, are you planning on releasing a chocolatey pkg for this new version?

**Comment by Cossey** [comment]
Yes, I just started the re-write the other day and I intend on releasing 2.0.0.0 in the next week. 

The core functionality has been completed. 

I will be updating the milestones soon with new features but these will mainly just be cosmetic like changing the ruler and notch colours. 

Please let me know if you have any further suggestions for this tool.

**Comment by Spunkie** [comment]
Awesome, I look forward to the release on chocolatey. 🎉

Edit: Actually I'll move the suggestion from this comment out into a separate issue for posterity's sake.

**Comment by Cossey** [comment]
Okay, version 2.0.0.0 is released here and were just waiting on the chocolatey package to be approved (but its submitted).
